src/mixer/basetrackplayer.cpp

```diff
@@ -478,6 +515,26 @@ void BaseTrackPlayerImpl::slotSetReplayGain(mixxx::ReplayGain replayGain) {
     }
 }
 
+void BaseTrackPlayerImpl::slotSetTrackColor(mixxx::RgbColor::optional_t color) {
+    m_pTrackColor->set(color ? static_cast<double>(*color) : kNoTrackColor);
+}
+
+void BaseTrackPlayerImpl::slotTrackColorChanged(double v) {
+    if (!m_pLoadedTrack) {
+        return;
+    }
+
+    mixxx::RgbColor::optional_t color = std::nullopt;
+    if (v != kNoTrackColor) {
+        auto colorCode = static_cast<mixxx::RgbColor::code_t>(v);
+        if (v != mixxx::RgbColor::validateCode(colorCode)) {
```

Comment:
you can use isValidCode()

Reply:
Ping!

Reply:
Sorry, I fixed it locally then it reset some changes due to the autoformatting issue and this must have been reverted to its previous state by accident.